packages/console-extension/src/index.ts

```diff
@@ -136,11 +138,13 @@ async function activateConsole(
   rendermime: IRenderMimeRegistry,
   settingRegistry: ISettingRegistry,
   launcher: ILauncher | null,
-  status: ILabStatus | null
+  status: ILabStatus | null,
+  sessionDialogs: ISessionContextDialogs | null
```

Comment:
you could just make this have a default value: `sessionDialogs = sessionContextDialogs`

Reply:
wait, no, that won't work if `null` is explicitly being passed in...

Reply:
If the supplied value is `null` I believe that wouldn't work